Makefile

```diff
@@ -93,34 +92,33 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
-##@ Build
+manifests: kustomize generate
+	$(KUSTOMIZE) build config/crd/ > $(DEPLOYDIR)/crd.yaml
+	$(KUSTOMIZE) build config/rbac/ > $(DEPLOYDIR)/rbac.yaml
+	cd config/manager && $(KUSTOMIZE) edit set image perconalab/percona-server-mysql-operator=$(IMG)
+	$(KUSTOMIZE) build config/manager/ > $(DEPLOYDIR)/operator.yaml
+	cat $(DEPLOYDIR)/crd.yaml $(DEPLOYDIR)/rbac.yaml $(DEPLOYDIR)/operator.yaml > $(DEPLOYDIR)/bundle.yaml
 
-build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager cmd/manager/main.go
-
-run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
-
-docker-build: test ## Build docker image with the manager.
-	docker build --build-arg GIT_COMMIT=${GIT_COMMIT} --build-arg BUILD_TIME=${BUILD_TIME} -t ${IMG} .
+##@ Build
 
-docker-push: ## Push docker image with the manager.
-	docker push ${IMG}
+build: test ## Build docker image with the manager.
+	./e2e-tests/build
 
 ##@ Deployment
 
-install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+install: manifests ## Install CRDs, rbac
+	kubectl apply -f $(DEPLOYDIR)/crd.yaml
+	kubectl apply -f $(DEPLOYDIR)/rbac.yaml
 
-uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl delete -f -
+uninstall: manifests ## Uninstall CRDs, rbac
+	kubectl delete -f $(DEPLOYDIR)/crd.yaml
+	kubectl delete -f $(DEPLOYDIR)/rbac.yaml
 
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default | kubectl apply -f -
+deploy: manifests ## Deploy operator
+	kubectl apply -f $(DEPLOYDIR)/operator.yaml
 
-undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/default | kubectl delete -f -
+undeploy: ## Undeploy operator
+	kubectl delete -f $(DEPLOYDIR)/operator.yaml
 
```

Comment:
how DEPLOYDIR content is generated? can we add a section here for it?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/12/commits/d58f477d1e45be115621da452a9b482e4aeb4cd6